app/models/conversation.rb

```diff
@@ -0,0 +1,28 @@
+class Conversation < ApplicationRecord
+  belongs_to(
+    :author,
+    class_name: User.name,
```

Comment:
You have to use a string like `"User"`, because if `Conversation` loads `User` and `User` loads `Conversation` the Rails autoloader breaks.

Reply:
Done in ab986b0